Makefile

```diff
@@ -67,6 +69,14 @@ $(RAW_DATA_FOLDER)/hp-mp2.semsim.tsv: $(RAW_DATA_FOLDER)/hp-mp-merged2.db $(RAW_
 	runoak -i $< similarity -p i,p $(TERM) @ $(TERM2) -O csv -o $@
 
 
+results/pheno_rustsim1.semsim1.tsv:
+	test -d results/ || mkdir -p results
+	runoak -i rustsim:sqlite:obo:phenio similarity -p i,p hand @ finger -O csv -o $@
```

Comment:
We need all the values, not just a handful. How long does this run ti generate the entire table?

Reply:
I tried to, but after more than 3 days we had only a few records :(

Reply:
Please discuss with the rustsim channel in Monarch, we absolutely need to prioritise this!